draco/core/containers.py

```diff
@@ -629,6 +629,87 @@ def conjugate(self):
         return self.index_map["stack"]["conjugate"]
 
 
+class SystemSensitivity(TODContainer):
+    """A container for holding a sensitivity dataset in time.
+
+    This should be averaged/collapsed in the stack/prod axis
+    to provide an overall summary of the system sensitivity.
+    Two datasets are available: the measured noise and the
+    estimated noise from the visibility weights.
+    """
+
+    _axes = ('freq', 'pol', 'time')
```

Comment:
You can remove the `time` entry from `_axes`. It's inherited (by some magic) from `TODContainer`